src/core/quicdef.h

```diff
@@ -397,6 +397,12 @@ QUIC_STATIC_ASSERT(
 //
 #define QUIC_DEFAULT_MAX_DATAGRAM_LENGTH        0xFFFF
 
+//
+// The maximum number of bytes of application data a server application can
+// send in a resumption ticket.
+//
+#define QUIC_MAX_RESUMPTION_APP_DATA_LENGTH     1000 
```

Comment:
I'm wondering if this actually makes sense to go in the public header instead, since it's a limit on what the app can do. They should know the limit, right?

Reply:
I agree. It should be more visible to consumers of the API.

Reply:
Let's move it to msquic.h